sematic/ui/packages/main/src/components/Health.tsx

```diff
@@ -0,0 +1,56 @@
+import { useContext, useEffect } from "react";
+import { useHttpClient } from "@sematic/common/src/hooks/httpHooks";
+import useAsync from "react-use/lib/useAsync";
+import SnackBarContext from "@sematic/common/src/context/SnackBarContext";
+import { MessageKind } from "@sematic/main/src/components/SnackBarProvider";
+
+type SingleStatus = {
+  healthy: boolean;
+  message: string;
+};
+
+
+type HealthReport = {
+  api: SingleStatus;
+  db: SingleStatus;  
+};
+
+export default function Health() {
+
+  const message = useFetchHealth();
+
+  const { setSnackMessage } = useContext(SnackBarContext);
```

Comment:
@chance-an I think that in the new Dashboard implementation, this message should be a banner instead of a toast.

Reply:
Yep, will keep that in mind for the new dashboard.